.golangci.yml

```diff
@@ -1,6 +1,8 @@
 run:
-  deadline: 10m
-  skip-dirs:
+  timeout: 10m
+
+issues:
+  exclude-dirs:
```

Comment:
The golangci-lint config fields changed a bunch.  If you run it on your local machine, make sure to update it to at least 1.58.